qiskit/circuit/library/standard_gates/x.py

```diff
@@ -1280,6 +1268,17 @@ def __new__(
             return gate
         return super().__new__(cls)
 
+    @deprecate_func(
+        additional_msg=(
+            "It is recommended to use :class:`.MCXGate` and let :class:`.HighLevelSynthesis` choose "
```

Comment:
The release-notes-reader in me is happy about the Sphinx links, but this message will only be displayed on the command line, so no need for annotations like `:class:` or `:func:` 🙂 In fact, personally I would even just write plain `MCXGate` (without the `'` bits) to make it easier to read, but this is up to you

Reply:
I like the "release-notes-reader in you" comment :). This is now fixed.